app/src/main/java/org/stacks/app/ui/secret/bottomsheet/ShareKeyBottomSheetFragment.kt

```diff
@@ -26,13 +26,18 @@ class ShareKeyBottomSheetFragment : BottomSheetDialogFragment() {
         ViewModelProvider(this).get(ShareKeyBottomSheetViewModel::class.java)
     }
 
+    var signUp: Boolean = false
```

Comment:
I believe you can pass this through the Fragment constructor, and only create it on demand.